### PR TITLE
Adding IoT Device Registration workflow

### PR DESCRIPTION
**Pull request by mheidir**
Please add this new workflow for BlueCat Gateway.

### CODE REVIEW

File: Community/iot_device_registration/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+IoT Device Registration
```
Comment (xiax) [inline Community/iot_device_registration/README.md]: Hi, we need the standard copyright in the readme.

Comment (michaelrn) [reply Community/iot_device_registration/README.md]: Hi Xiao, I have added the standard copyright to the README.

File: Community/iot_device_registration/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# IoT Device Registration
```
Comment (delmeherbert) [inline Community/iot_device_registration/README.md]: Hi Muhammed, the README looks good but I ask that you add the copyright date and the Apache license info at the top of the page for consistency with all other READMEs. Please have a look at the other README files in workflows in the Community folder.

File: Community/iot_device_registration/__init__.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
```
Comment (delmeherbert) [inline Community/iot_device_registration/__init__.py]: Please change all BlueCat copyright dates to 2019. Also in iotdeviceregistration_page.css, iotdeviceregistration_page.js, and iotdeviceregistration_page.html.

File: Community/iot_device_registration/README.md (hunk 2 above)
Comment (michaelrn) [reply Community/iot_device_registration/README.md]: README updated

File: Community/iot_device_registration/__init__.py (hunk 3 above)
Comment (michaelrn) [reply Community/iot_device_registration/__init__.py]: Copyright dates changed

**Pull request merged**